rpm/podman.spec

```diff
@@ -187,7 +189,16 @@ when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 Summary: Metapackage for setting up %{name} machine
 Requires: %{name} = %{epoch}:%{version}-%{release}
 Requires: gvisor-tap-vsock
-Requires: qemu
+%if %{defined qemu}
+%ifarch aarch64
+Requires: qemu-system-aarch64-core
+%else
+Requires: qemu-system-x86-core
```

Comment:
Not sure it is too over-simplified? There also MIPS/RISC/s390x arches in Fedora?

I understand podman-machine is likely supported only for x86_64 and aarch64. In such a case the safer conditioning would be something like:

```
%ifarch aarch64
Requires: qemu-system-aarch64-core
%else
%ifarch x86_64
Requires: qemu-system-x86-core
%endif
```

I.e. it won't add any dependency on unsupported arch in case a new arch is added to ExclusiveArch?

Reply:
fixed. adjusted the conditionals a little to get rid of the `%else after %else` rpmlint error. Also included an empty `%check` section in a separate commit to silence rpmlint warning.